grub.cfg-recovery

```diff
@@ -0,0 +1,40 @@
+set default=0
+set timeout=3
+
+insmod part_gpt
+
+if [ -s $prefix/grubenv ]; then
+  load_env
+fi
+
+# FIXME: find a better way? this ensures that someone instructed snapd
+#        to go into "install" mode it actually does that
+if [ "$snap_mode" = "install" ]; then
+    set default=1
```

Comment:
Discussed in London that these don't need to be menu entries at all: if the chooser interaction is expected to be entirely via the initramfs, this could execute the boot commands inline instead of driving a menu selection.

Reply:
Yes, mostly for convenience today to see things flipping from "recovery" -> "normal" (a bit of a debug aid).